pkg/envoy/rbac/policy.go

```diff
@@ -94,7 +94,7 @@ func (p *Policy) Generate() (*xds_rbac.Policy, error) {
 					if err != nil {
 						return nil, errors.Errorf("Error parsing destination port value %s", andPermissionRule.Value)
 					}
-					portPermission := getPermissionDestinationPort(uint32(port))
+					portPermission := GetPermissionDestinationPort(uint32(port))
```

Comment:
Is this semantically - get the allowed port?

Reply:
This is a helper, that takes in a port and returns a Permission type corresponding to the DestinationPort.